core/domain/exp_jobs_one_off.py

```diff
@@ -523,14 +523,3 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
-
-
-class DeleteStateIdMappingModelsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Do we have to delete the tests related to this job?

Reply:
Missed that! Done.